### DISCUSSION TITLE
Change Download Failed to Report Link Failed On

### DISCUSSION THREAD

**Discussion by ajax2020-vik** [aclap-dev/video-downloadhelper#920] [Help and Q&A]
### Your question

The newest version supplies a very generic error of the download failed; however, if you are performing a number of downloads, you have to sift through to find out which one failed. This is not the behavior of the older versions.

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"51d2f70cf9254dd5"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\galex\\dwhelper",
    "concurrent_downloads_max": 3,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": false,
    "open_count_store": 70,
    "successfull_dl": 56,
    "never_show_successfull_dl_message": true
}

**Comment by ThomasBecker42** [comment]
I have the same problem. When downloading multiple videos and an "Download failed" I have no chance to see which one it was.
With the old UI/Version I was able to see the name of the video.
Firefox 127.0.2 (64-Bit)
Version 9.0.2.12 (stable), target mozilla, locale de.

**Comment by GrampaWildWilly** [comment]
See my response in #948.
